src/bournemouth/app.py

```
@@ -15,7 +15,7 @@
     from sqlalchemy.ext.asyncio import AsyncSession
 
 from .auth import AuthMiddleware, LoginResource
-from .chat_service import stream_answer as default_stream_answer
+from .chat_service import StreamFunc, stream_answer as default_stream_answer
```

Pylint (3.3.7)

[convention] Import "from .chat_service import StreamFunc, stream_answer as default_stream_answer" should be placed at the top of the module

(C0413)